helm/alfresco-content-services/templates/config-repository.yaml

```diff
@@ -39,11 +39,7 @@ data:
       -Dshare.host={{ tpl (.Values.externalHost | default (printf "%s-share" (include "content-services.shortname" .))) $ }}
       -Dshare.port={{ tpl (.Values.externalPort | default .Values.share.service.externalPort | toString) $ }}
       -Dalfresco_user_store.adminpassword={{ .Values.repository.adminPassword | default "209c6174da490caeb422f3fa5a7ae634" }}
-      {{- if and 
-        (not (index .Values "alfresco-search" "enabled"))
-        (not (index .Values "alfresco-elasticsearch-connector" "enabled"))
-        (not (index .Values "alfresco-search" "external" "host"))
-      }}
+      {{- if and (not (index .Values "alfresco-search" "enabled")) (not (index .Values "alfresco-elasticsearch-connector" "enabled")) (not (index .Values "alfresco-search" "external" "host")) }}
```

Comment:
Can I know what is this change doing?

Reply:
unfolding the template seems to fix the problem with fluxcd's helm-operator one the integration pipeline. That's the issue you had on the opsexp-1767 branch

Reply:
Thanks for explaining this.